WGLC changes in Privacy Considerations section. (#118)
mlswg/mls-architecture: draft-ietf-mls-architecture.md

of the message stream, but does allow it to attack forward security to a limited
extent. This threat can be mitigated by having initial keys expire.

Initial keying material (KeyPackages) using the `basic` Credential type
is more vulnerable to replacement by a malicious or compromised DS, as
there is no built-in cryptographic binding between the identity and the

> public key of the client. 

Reviewer: ```suggestion
public key of the client.
```

> **RECOMMENDATION:**
> Prefer a Credential type in KeyPackages which includes a strong
> cryptographic binding between the identity and its key (for example
> the `x509` Credential type). When using the `basic` Credential type

> > take extra care to verify the identity (typically out-of-band).

Reviewer 2: If you look at most E2E-encrypted messaging applications today (e.g., Signal, WhatsApp, Wire), there's no binding of identity material to key material that is safe from DS manipulation, aside from manual verification.  If these systems were to move to MLS without inventing a new identity system, `basic` would be the obvious way to do it.

clients know who is in the group at all times. This means that - if all Members
of the group and the Authentication Service are honest - no other parties than
the members of the current group can read and write messages protected by the

> protocol for that Group. Note that when BasicCredential is used, the
> Authentication Service also needs an out-of-band mechanism to verify the
> identity asserted in the Credential.

Reviewer 2: ```suggestion
protocol for that Group. Note that when a `basic` Credential is used, the
Authentication Service also needs an out-of-band mechanism to verify the
identity asserted in the Credential.
```

Beware though, the link between the cryptographic identity of the Client and the
real identity of the User is important. With some Authentication Service
designs, a private or centralized authority can be trusted to generate or
validate signature keypairs used in the MLS protocol. This is typically the
case in some of the biggest messaging infrastructures.

Details about how to verify the identity of a client depend on the MLS
Credential type used. For example, cryptographic verification of credentials
can be largely performed autonomously on the clients for the `x509`
Credential type. In contrast, when MLS clients use the `basic` Credential type,
a larger degree of trust must be placed in a (likely) centralized

> authentication resource.

Reviewer 2: ```suggestion
authentication resource, or on out-of-band processes such as manual verification.
```
